eyelid-match-ops/Cargo.toml

```diff
@@ -45,6 +46,7 @@ eyelid-test.workspace = true
 
 lazy_static.workspace = true
 rand.workspace = true
+rand_distr.workspace = true
```

Comment:
Production dependencies are automatically included in dev builds, so we can delete all 3 of these.

(I think we made then prod dependencies in previous PRs, but forgot to delete them here.)